src/Microsoft.AspNet.SignalR.SqlServer/TaskAsyncHelper.cs

```diff
@@ -0,0 +1,1194 @@
+// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
```

Comment:
Is this file actually used? Last I checked there were only a couple simple uses of the TaskAsyncHelper that should be easy to replace.